addons/survey/static/src/js/survey_quick_access.js

```diff
@@ -54,10 +72,10 @@ publicWidget.registry.SurveyQuickAccessWidget = publicWidget.Widget.extend({
 
     _submitCode: function () {
         var self = this;
-        this.$('.o_survey_error > span').addClass("d-none");
+        this.$('.o_survey_error > span').addClass('d-none');
```

Comment:
Not sure we need to go as far as to convert quotes ^^
Both are equally accepted in guidelines.
(It's still good to have consistency in the same file, but we usually don't modify existing code).

It makes it a bit hard to see the actual changes here below as it creates a bit of noise.